Automattic/hostmgr
# Add support for Apple Silicon VMs

To test:

1. On this branch, run `make run-vm-create-debug` to download the latest macOS restore image and create a VM from it.
2. Once the VM is successfully created, run `make run-helper-debug` to start up the XPC helper in debug mode, then click "Start VM" to launch your new VM.
3. To test packaging the VM for upload, run `swift run hostmgr vm package --name test` – note that the file (`test.aar`) is in your `$TMPDIR`
